src/dateAndTime.js

```diff
@@ -100,6 +100,15 @@ const humanFriendlyTime = (date = new Date(), locales = "en-US") => {
   return date.toLocaleTimeString(locales);
 };
 
+/**
+ * Converts date into Unix Timestamp formate.
+ * @param {Date} date date
+ * @returns {Number} In unix timestamp formate.
+ */
+const toUnixTimeStamp = (date = new Date()) => {
```

Comment:
This function doesn't work.

It throws the following error:

_Uncaught ReferenceError: toTimeStamp is not defined_